docs/prepare-osx-devbox.md

```diff
@@ -0,0 +1,29 @@
+# Prepare your development environment using MacOS X
+
+We are open to developers on OSX working on this repository. We are asking macOS users to setup GNU utils on their machines.
```

Comment:
sorry to be a pain, would you mind wrapping at 80 for markdown?  We do everywhere else.

Reply:
NP good to know :) I’ll push an update